docs/content/how-to/create-azure-cluster.md

```diff
@@ -0,0 +1,25 @@
+ # Creating an Azure cluster
+
+This document describes how to set up an Azure cluster with Hypershift.
+
+## Prerequisites
+
+In order to authenticate with Azure, an Application must be created through the web portal. Follow [this guide](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) to create one.
+
+Afterwards, create a credentials file that looks like this:
+
+```
+AZURE_SUBSCRIPTION_ID: <your_subscription_id>
+AZURE_TENANT_ID: <your_tenant_id>
+AZURE_CLIENT_ID: <your_client_id>
+AZURE_CLIENT_SECRET: <your_client_secret>
+```
+
+## Creating the cluster
+
+After the credentials file was set up, creating a cluster is a simple matter of invoking the `hypershift` cli:
+
+
+```
+hypershift cluster azure --pull-secret <pull_secret_file> --name <cluster_name> --azure-creds <path_to_azure_credentials_file>
```

Comment:
`--name` is the name of the cluster which together with the infra id is used as input for naming the resources created in Azure.

> can i specify the parent resource group to nest the cluster within?

No, currently it will always create a non-nested resource group. From the looks of it, the command you pasted assumes that all these infra resources already exist?

> do we want to expose location?

It is already exposed, I've updated the doc to include the parameter.

> are we tagging the resources we create?

Right now we always create a resource group in which everything else is created. This makes the resource group the infra boundary, so tagging isn't really needed. This is again derived from how the installer for selfhosted does this. If we added an option to use an existing resource group and not create a new one, tagging would make sense.

> for bring your own vnet, we may want to let you specify the id of vnet and its resource group.

Yeah, makes sense. I'll keep "bring your own vnet" as a follow-up.